auth/api/iam/authorized_code.go

```diff
@@ -38,7 +39,7 @@ func (a authorizedCodeFlow) handleAuthzRequest(params map[string]string, session
 
 	// Render HTML
 	buf := new(bytes.Buffer)
-	// TODO: Support multiple languages
+	// TODO: Support multiple languages -- but how with the template path hardcoded prior to the request?
```

Comment:
this authorizedCodeFlow type will probablyt go away, it's a reminiscant of the previous, complicated, OAuth2 design I made earlier